dss/api/health.py

```diff
@@ -33,10 +40,10 @@ def _get_dynamodb_status():
     """Checks dynamoDB table status, tables are explicitly specified within the function"""
     db_status = True
     stage = os.getenv("DSS_DEPLOYMENT_STAGE")
-    ddb_tables = ["dss-async-state-{}".format(stage),
-                  "dss-subscriptions-v2-aws-{}".format(stage),
-                  "dss-subscriptions-v2-gcp-{}".format(stage)
-                  ]
+    service_tags = {"Key": "service", "Values": ["dss"]}
+    resource_list = get_resource_by_tag(resource_string='dynamodb:table', tag_filter=service_tags)
```

Comment:
I think the same effect can be achieved replacing `dynamodb:table` with `dynamodb`.

Reply:
I'm going to keep it as tables to ensure the resource filter is kept to tables in this function.